### PR TITLE
Preview Release of module 'Az.ADDomainServices'

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by isra-fel** [comment]
I notice the "General availability" in PR title. We are not going to GA (1.0.0) this module at first release, are we?

**Comment by BethanyZhou** [comment]
It's a preview release.

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/ADDomainService/readme.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          "204": {
+            "description": "HTTP 204 (Not Content) should be used if the resource does not exist and the request is well formed."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+          }
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply src/ADDomainService/readme.md:76]: Thanks for your suggestion~

File: src/ADDomainService/examples/New-AzADDomainService.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+### Example 1: Create new ADDomainService
+```powershell
+PS C:\> $NewAdDomain = New-AzADDomainServiceReplicaSetObject -Location westus -SubnetId /subscriptions/********-****-****-****-**********/resourceGroups/yishitest/providers/Microsoft.Network/virtualNetworks/aadds-vnet/subnets/default
+New-AzADDomainService -name youriADdomain -ResourceGroupName youriAddomain -DomainName youriAddomain.com -ReplicaSet $NewAdDomain -debug
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply src/ADDomainService/examples/New-AzADDomainService.md:4]: LGTM,thanks.

File: src/ADDomainService/readme.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  - $(repo)/specification/domainservices/resource-manager/Microsoft.AAD/stable/2020-01-01/domainservices.json
+# If the swagger has not been put in the repo, you may uncomment the following line and refer to it locally
+# - (this-folder)/relative-path-to-your-swagger 
+
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply src/ADDomainService/readme.md:41]: Got it.Thanks~

File: src/ADDomainService/custom/New-AzADDomainService.ps1 [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+The Create Domain Service operation creates a new domain service with the specified parameters.
+If the specific service already exists, then any patchable properties will be updated and any immutable properties will remain unchanged.
+.Description
+The Create Domain Service operation creates a new domain service with the specified parameters.
+If the specific service already exists, then any patchable properties will be updated and any immutable properties will remain unchanged.
+.Example
```
Comment (dolauli) [inline src/ADDomainService/custom/New-AzADDomainService.ps1:23]: Please clean empty examples

File: src/ADDomainService/custom/Update-AzADDomainService.ps1 [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+<#
+.Synopsis
+The Update Domain Service operation can be used to update the existing deployment.
+The update call only supports the properties listed in the PATCH body.
+.Description
+The Update Domain Service operation can be used to update the existing deployment.
+The update call only supports the properties listed in the PATCH body.
+.Example
+PS C:\> {{ Add code here }}
```
Comment (dolauli) [inline src/ADDomainService/custom/Update-AzADDomainService.ps1:24]: The same as above

**Review by dolauli — CHANGES_REQUESTED** [review]
Generally looks good to me. And some minor changes are required.

**Comment by BethanyZhou** [comment]
Thanks @dolauli 's advices, have updated codes.

**Review by dolauli — APPROVED** [review]
Looks good to me.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**